packages/client/mutations/connections/getUserTasksConn.ts

```diff
@@ -0,0 +1,7 @@
+import {ConnectionHandler, RecordProxy} from 'relay-runtime'
+
+const getUserTasksConn = (viewer: RecordProxy, userIds: string[] | null, teamIds: string[] | null) =>
+
+  ConnectionHandler.getConnection(viewer, 'UserColumnsContainer_tasks', {userIds, teamIds})
```

Comment:
this is preparing us for the next iteration. now, we can handle any combination of userIds & teamIds.